receiver/etwreceiver/config.go

```diff
@@ -4,9 +4,39 @@
 package etwreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/etwreceiver"
 
 import (
+	"fmt"
+	"strings"
+
 	"go.opentelemetry.io/collector/component"
 )
 
+type TraceLevel int
+
+const (
+	TraceLevelCritical TraceLevel = iota + 1
+	TraceLevelError
+	TraceLevelWarning
+	TraceLevelInformation
+	TraceLevelVerbose
+)
+
+func TraceLevelFromString(level string) (TraceLevel, error) {
+	switch strings.ToLower(level) {
```

Comment:
Just so we don't fail on mistakes like this.

```suggestion
	switch strings.ToLower(strings.TrimSpace(level)) {
```